eng/pipelines/runtime-linker-tests.yml

```diff
@@ -7,13 +7,45 @@ trigger:
   branches:
     include:
     - master
+    - dev/infrastructure
     - release/*.*
+  paths:
+    include:
+    - '*'
+    - docs/manpages/*
+    exclude:
+    - eng/Version.Details.xml
+    - .github/*
+    - docs/*
+    - CODE-OF-CONDUCT.md
+    - CONTRIBUTING.md
+    - LICENSE.TXT
+    - PATENTS.TXT
+    - README.md
+    - SECURITY.md
+    - THIRD-PARTY-NOTICES.TXT
 
 pr:
   branches:
     include:
     - master
+    - dev/infrastructure
     - release/*.*
+  paths:
+    include:
+    - '*'
+    - docs/manpages/*
```

Comment:
I think this pipeline can just exclude all .MD files no need to run them when docs/manpages change. That is only relevant to the installer and this pipeline doesn’t even build the installer.

Reply:
that's true but on the other hand I find it easier to understand if we keep this and the main runtime.yml 100% the same :)

Reply:
I agree on that, but I think the point is to use as little resources as possible, so that would be a valid reason for it to differ.

Reply:
Ok, changed it.

Reply:
Thanks 😄